microsoft-edge/devtools-guide-chromium/layers/layers-tool.md

```diff
@@ -0,0 +1,17 @@
+---
+title: Layers tool
+description: In Microsoft Edge DevTools, the Layers tool represents the webpage in a 3D space.  The Layers tool has largely been superseded by the 3D View tool.
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.prod: microsoft-edge
+ms.technology: devtools
+ms.date: 02/15/2022
+---
+# Layers tool
+
+The **Layers** tool represents the webpage in a 3D space.  The **Layers** tool has largely been superseded by the **3D View** tool.
+
+See [Navigate z-index, DOM, and layers using the 3D View tool](../3d-view/index.md).
```

Comment:
We should link directly to this section instead: https://docs.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/3d-view/#composited-layers